app/public/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "Contacto",
-  "version": "0.0.3",
+  "name": "Plivo CX Service",
+  "version": "1.0.4",
```

Comment:
shouldn't it be 1.0.0?

Reply:
there is already a version of 1.0.3 in chrome webstore. That update is not pushed to main branch somehow. So updated to 1.0.4

Reply:
okay